astropy/cosmology/parameters.py

```diff
@@ -150,4 +144,5 @@
                "Table 1 (WMAP + BAO + SN ML).")
 )
 
-available = tuple(k for k in locals() if not k.startswith('_'))
+# If new parameters are added, this list must be updated
```

Comment:
If one had a class, it would be quite trivial to have it have a registry as well. But probably overkill, and then it gets even further away from the primary purpose of this PR... The logic for my suggestion was that it keeps the code more similar but still allows for not having to delete imports.